src/main/java/com/google/devtools/build/lib/analysis/RunfilesSupport.java

```diff
@@ -453,6 +453,20 @@ public static RunfilesSupport withExecutable(
         computeActionEnvironment(ruleContext));
   }
 
+  /**
+   * Creates and returns a {@link RunfilesSupport} object for the given rule and executable. This
+   * version discards all arguments. Only use this for Incompatible Target Skipping.
```

Comment:
Can you link this more precisely to a specific file or source javadoc? 

This comment comes from the hard-learned lesson that even specific concepts can be fuzzy to devs who haven't focused specifically on them. Precise links are always great for that.

Reply:
I can't verify that the link renders correctly, but I added a link to https://docs.bazel.build/versions/master/platforms.html#skipping-incompatible-targets

Would you prefer a link to the `site/docs/platforms.md` file instead?

Reply:
That's fine, thanks. Just anything to connect the breadcrumbs.